src/components/heading/heading.js

```diff
@@ -120,6 +120,8 @@ class Heading extends React.Component {
 
     return (
       <Link
+        // this event allow en element to be focusable on cick event on IE
+        onMouseDown={ e => e.currentTarget.focus() }
```

Comment:
Would using `:active` pseudo-class not achieve the same effect? Instead of faking a blur.

https://developer.mozilla.org/en-US/docs/Web/CSS/:active

Reply:
I tried the solution but Internet Explorer won't cooperate :/ 
This is the one solution I found for the Internet Explorer. All because there is an anchor and IE doesn't want to setup focus on it while clicking.